src/web/components/Card/components/CardAge.tsx

```diff
@@ -9,32 +9,47 @@ import ClockIcon from '@frontend/static/icons/clock.svg';
 
 import { makeRelativeDate } from '@root/src/web/lib/dateTime';
 import { decidePillarLight } from '@root/src/web/lib/decidePillarLight';
+import { space } from '@guardian/src-foundations';
+import { until } from '@guardian/src-foundations/mq';
 
-const ageStyles = (design: Design, pillar: Theme) => css`
-	${textSans.xsmall()};
-	color: ${design === Design.Live ? decidePillarLight(pillar) : neutral[60]};
+const ageStyles = (
+	design: Design,
+	pillar: Theme,
+	isFullCardImage?: boolean,
+) => {
+	// This is annoying but can't apply SVG color otherwise
+	const smallImageSvgColor =
+		design === Design.Live ? decidePillarLight(pillar) : neutral[46];
```

Comment:
there's some similar logic for color on line 26 which we could pull out like this.

I also spot some additional logic on line 39 which might make sense to include here so it's in one place?